extensions/infinispan-client/deployment/src/main/java/io/quarkus/infinispan/client/deployment/devservices/InfinispanDevServiceProcessor.java

```diff
@@ -283,14 +283,22 @@ public QuarkusInfinispanContainer(String clientName, InfinispanDevServicesConfig
                 return " -c " + userConfigFile;
             }).collect(Collectors.joining())).orElse("");
 
+            if (config.tracing.isPresent()) {
+                log.info(
+                        "This option won't work from 15.0. You have to use `configFiles` and provide an json, xml or yaml file, as this example:");
```

Comment:
I think we need to make the message more actionable. This is a start but we also need to be more precise about this `configFiles` option. I have no idea how to pass it to the Dev Services for Infinispan?

```suggestion
                        "Starting with Infinispan 15.0, the <full name of the property> configuration property is not supported anymore. You have to use `configFiles` and provide a JSON, XML or YAML file, for instance:");
```

Reply:
Also, this should probably be a warning, not an info.